inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/elasticsearch/EsSinkContext.java

```diff
@@ -17,13 +17,7 @@
 
 package org.apache.inlong.sort.standalone.sink.elasticsearch;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.LinkedBlockingQueue;
-import java.util.concurrent.atomic.AtomicLong;
+import com.alibaba.fastjson.JSON;
```

Comment:
There are too many security risks in this package, could you replace it with the Gson?

Reply:
change to com.fasterxml.jackson.databind.ObjectMapper